### PR TITLE
DDCE-4866 Add tests for new change role page and navigation

### PR DESCRIPTION
**Pull request by mywyau**

### CODE REVIEW

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/ChangeImporterRoleSteps.scala [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
```
Comment (gastonracui) [inline src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/ChangeImporterRoleSteps.scala:2]: 2024...

Comment (gastonracui) [reply src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/ChangeImporterRoleSteps.scala:2]: and also check at other files

**Review by gastonracui — CHANGES_REQUESTED** [review]

**Comment by gastonracui** [comment]
check PR recommendations as well

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/ChangeImporterRoleSteps.scala (hunk 1 above)
Comment (mywyau) [reply src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/ChangeImporterRoleSteps.scala:2]: done

**Review by gastonracui — APPROVED** [review]

**Pull request merged**